be/src/olap/tablet_meta.cpp

```diff
@@ -1432,13 +1432,13 @@ std::shared_ptr<roaring::Roaring> DeleteBitmap::get_agg(const BitmapKey& bmk) co
                 val->bitmap |= reinterpret_cast<DeleteBitmapAggCache::Value*>(
                                        DeleteBitmapAggCache::instance()->value(handle2))
                                        ->bitmap;
-                DeleteBitmapAggCache::instance()->release(handle2);
                 VLOG_DEBUG << "get agg cache version=" << start_version
                            << " for tablet=" << _tablet_id
                            << ", rowset=" << std::get<0>(bmk).to_string()
                            << ", segment=" << std::get<1>(bmk);
                 start_version += 1;
             }
+            DeleteBitmapAggCache::instance()->release(handle2);
```

Comment:
should check handle2 is not nullptr?

Reply:
No, the `release` function has already handled nullptr.

Reply:
@mymeiyi `LRUCache::release` handed nullptr, but `ShardedLRUCache::release` not, 😑